docs/content/practices/emulator_setup.md

```diff
@@ -48,19 +48,19 @@ skin.name=320x480
 disk.dataPartition.size=8G
 ```
 
-Pay your attention that we disabled:
+Pay attention to what we have disabled:
 
 * Accelerometer
 * Audio input/output
 * Play Store
 * Sensors:Accelerometer, Humidity, Pressure, Light
 * Gyroscope
 
-We don't really need them for our tests run. It also may improve our tests performance, because there are no background
-operations related to that tasks.
+We don't really need them for our test runs. It also may improve our tests performance, because there are no background
+operations related to those tasks.
 
-After that, you can run your emulator by `avd manager`, which is a part of android sdk manager. After your device
-creation, you need change default generated ini file to custom one. You may see an example below:
+After that, you can run your emulator via `avd manager`, which is part of the android sdk manager. After creating the emulator, you need to switch the default generated `ini` file to the custom one we defined previously. 
```

Comment:
I rephrased this. I believe this is what is intended after reading the code in the script